.changeset/pretty-toes-tan.md

```diff
@@ -0,0 +1,31 @@
+---
+'@leafygreen-ui/text-area': major
+---
+
+[LG-4144](https://jira.mongodb.org/browse/LG-4144)
+
+1. `FormField` styling changes apply to `TextArea`. [See style changes here](https://github.com/mongodb/leafygreen-ui/blob/main/packages/form-field/CHANGELOG.md#102)
+
+2. A default `errorMessage` of `'This input needs your attention'` will render below text area when state is invalid.
```

Comment:
This feels strange to me. Where is this coming from?

Reply:
@TheSonOfThomp yes, this is a new spec from design so the error icon doesn't show up by itself